Makefile

```
@@ -40,11 +40,17 @@ lint: lint-proto lint-go  ## Lint code and protos
 .PHONY: lint-go
 lint-go: $(BIN)/golangci-lint
 	$(BIN)/golangci-lint run --modules-download-mode=readonly --timeout=3m0s ./...
+	$(BIN)/golangci-lint fmt --diff
 
 .PHONY: lint-proto
 lint-proto: $(BIN)/buf
 	$(BIN)/buf lint
 
+.PHONY: lintfix
+lintfix:
```

To be consistent with the other targets:

```suggestion
.PHONY: lint-fix
lint-fix:
```